akka-http-tests/src/test/resources/reference.conf

```diff
@@ -6,4 +6,5 @@ akka {
     default-dispatcher.throughput = 1
   }
   stream.materializer.debug.fuzzing-mode = on
+  strict-entity-max-bytes = 9223372036854775807
```

Comment:
no need to repeat this in http-tests or docs, the default one from http-core will be loaded already.